build.sbt

```diff
@@ -491,19 +492,37 @@ lazy val `persistence-cassandra` = (project in file("persistence-cassandra"))
     )
   ) configs (MultiJvm)
 
-lazy val `persistence-jdbc` = (project in file("persistence-jdbc"))
-  .settings(name := "lagom-javadsl-persistence-jdbc")
-  .dependsOn(persistence % "compile;test->test", `persistence-javadsl` % "compile;test->test")
+
+lazy val `persistence-jdbc-core` = (project in file("persistence-jdbc/core"))
+  .settings(name := "lagom-persistence-jdbc-core")
```

Comment:
for persistence I didn't bother about naming the common module `-core`, not sure what is best